### PR TITLE
Get Target Details structure fixed for new SDK (3.3.16)

### PR DESCRIPTION
**Pull request by alikdolg**

### CODE REVIEW

File: akeyless/data_source_target_details.go [hunk 1]
```diff
@@ -109,38 +109,38 @@ func dataSourceGetTargetDetailsRead(d *schema.ResourceData, m interface{}) error
 	if rOut.Value == nil {
 		return fmt.Errorf("can't get value")
 	}
-	if rOut.Value.Username != nil {
-		err = d.Set("username", *rOut.Value.Username)
+	if rOut.Value.SshTargetDetails.Username != nil {
```
Comment (tuvia-akeyless) [inline akeyless/data_source_target_details.go:112]: are you sure it is SSH target? how?

**Review by tuvia-akeyless — COMMENTED** [review]
need to bump the version (in "version" file)

File: akeyless/data_source_target_details.go (hunk 1 above)
Comment (alikdolg) [reply akeyless/data_source_target_details.go:112]: I'm not sure - just found object with same parameters and I don't know how to check it

Comment (tuvia-akeyless) [reply akeyless/data_source_target_details.go]: i inserted commit to fix it

**Review by idanmantin — APPROVED** [review]

**Pull request merged**